src/logger/DebugLogger.ts

```diff
@@ -0,0 +1,103 @@
+import * as debug from "debug";
+import { Logger } from "./Logger";
+import { QueryRunner } from "../";
+import { PlatformTools } from "../platform/PlatformTools";
+
+/**
+ * Performs logging of the events in TypeORM via debug library.
+ */
+export class DebugLogger implements Logger {
+    private debugQuery = debug("typeorm:query");
+    private debugQueryError = debug("typeorm:queryError");
```

Comment:
> typeorm:queryError

queryError looks rough, is there naming convention in `debug`? Maybe better name it `typeorm:query:error` or `typeorm:query-error` ?

Reply:
https://github.com/visionmedia/debug#conventions